product/components/ProductCard.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import faker from "faker";
+
+import ProductCard from "./ProductCard";
+
+export default {title: "ProductCard"};
+
+export const noData = () => (
+  <ProductCard
+    add={() => {}}
+    product={{
+      id: faker.random.uuid(),
+      category: faker.commerce.department(),
+      image: faker.image.food(),
```

Comment:
```suggestion
      image: `https://placehold.it/320x240`,
```
Las imagenes de faker estan buenisimas pero tardan banda en cargar jaja